app/modules/wallet/addresses/addresses.controller.ts

```diff
@@ -0,0 +1,56 @@
+import * as restify from 'restify';
+import { InternalServerError, NotFoundError } from 'restify-errors';
+import IController from '../../../interfaces/utils/IController';
+import IRequest from '../../../interfaces/utils/IServer';
+import WalletAddress from '../../../models/wallet_address.model';
+import { IWallet } from '../../../interfaces/models';
+
+export default class WalletsController implements IController {
+  public async put(req: IRequest, res: restify.Response, next: restify.Next) {
+    try {
+      const walletAddress = await WalletAddress.create({
+        chain: req.body.chain,
+        address: req.body.address,
+        name: req.body.name,
+        wallet_id: req.params.wallet
+      });
+
+      return res.send({
+        walletAddress
+      });
+    } catch (e) {
+      console.log(e);
+      throw new InternalServerError(e);
+    }
+    return next();
+  }
+
+  public async get(req: IRequest, res: restify.Response, next: restify.Next) {
+    try {
+      const walletAddresses = await WalletAddress.findAll({
+        where: { wallet_id: req.params.wallet }
+      });
+      if (!walletAddresses || walletAddresses.length === 0) {
```

Comment:
No need to return any error if addresses doesn't exist yet. Just return `walletAddresses` as it'll be empty array if not found anything.